dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb

```diff
@@ -25,6 +25,7 @@ def index
     per = params[:per] || params[:limit] || 50
     user_query = params[:user_q]
 
+    # Retrieve all peer reviews
```

Comment:
I know you didn't add this, but why is it necessary to query for all `PeerReview`s?

Reply:
I see, it may be possible here to optimize and only query for all peer reviews if there are no filters. I'll leave it up to you if you want to change things here

Reply:
I think I will leave it until I have a better handle on the future uses of this dashboard. The current scenario: the entire dashboard and all reviews load, and the user can (only) then filter the reviews based on a course/course unit, etc.

Reply:
Yeah this does seem questionable, but at least I believe this route is admin only.